src/include/com/amazonaws/kinesis/video/webrtcclient/Stats.h

```diff
@@ -0,0 +1,322 @@
+ * object
+ */
+typedef struct {
+    CHAR dscp[MAX_STRING_LENGTH + 1]; //!< DSCP String
+    UINT64 totalNumberOfPacketsSent; //!< Number of packets sent
+} DscpPacketsSentRecord, *PDscpPacketsSentRecord;
+
+/**
+ * @brief RtcIceCandidatePairStats Stats related to the local-remote ICE candidate pair
+ *
+ * Reference: https://www.w3.org/TR/webrtc-stats/#candidatepair-dict*
+ */
+
+typedef struct {
+    CHAR transportId[MAX_STRING_LENGTH + 1]; //!< ID of object that was inspected for RTCTransportStats
+    CHAR localCandidateId[MAX_CANDIDATE_ID_LENGTH + 1]; //!< Local candidate that is inspected in RTCIceCandidateStats
+    CHAR remoteCandidateId[MAX_CANDIDATE_ID_LENGTH + 1]; //!< Remote candidate that is inspected in RTCIceCandidateStats
+    ICE_CANDIDATE_PAIR_STATE state; //!< State of checklist for the local-remote candidate pair
+    BOOL nominated; //!< Flag is TRUE if the agent is a controlling agent and FALSE otherwise. The agent role is based on the
+                    //!< STUN_ATTRIBUTE_TYPE_USE_CANDIDATE flag
+    NullableUint32 circuitBreakerTriggerCount; //!< Represents number of times circuit breaker is triggered during media transmission
+                                       //!< It is undefined if the user agent does not use this
+    UINT32 packetsDiscardedOnSend; //!< Total number of packets discarded for candidate pair due to socket errors,
+    UINT64 packetsSent; //!< Total number of packets sent on this candidate pair;
+    UINT64 packetsReceived; //!< Total number of packets received on this candidate pair
+    UINT64 bytesSent; //!< Total number of bytes (minus header and padding) sent on this candidate pair
+    UINT64 bytesReceived; //!< Total number of bytes (minus header and padding) received on this candidate pair
+    UINT64 lastPacketSentTimestamp; //!< Represents the timestamp at which the last packet was sent on this particular
+                                    //!< candidate pair, excluding STUN packets.
+    UINT64 lastPacketReceivedTimestamp; //!< Represents the timestamp at which the last packet was sent on this particular
+                                        //!< candidate pair, excluding STUN packets.
+    UINT64 firstRequestTimestamp; //!< Represents the timestamp at which the first STUN request was sent on this particular candidate pair.
+    UINT64 lastRequestTimestamp; //!< Represents the timestamp at which the last STUN request was sent on this particular candidate pair.
+                                 //!< The average interval between two consecutive connectivity checks sent can be calculated:
+                                 //! (lastRequestTimestamp - firstRequestTimestamp) / requestsSent.
+    UINT64 lastResponseTimestamp; //!< Represents the timestamp at which the last STUN response was received on this particular candidate pair.
+    DOUBLE totalRoundTripTime; //!< The sum of all round trip time (seconds) since the beginning of the session, based
```

Comment:
Generally, I discourage usage of double or float types measuring time for example. Is this spec-ed? The entire SDK is based on the standard time units - 100ns and all our time and period measurements are in UINT64

Reply:
Same for other instances of double